b2/account_info.py

```diff
@@ -0,0 +1,318 @@
+######################################################################
+#
+# File: b2/account_info.py
+#
+# Copyright 2016 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+import json
+import os
+import stat
+from abc import (ABCMeta, abstractmethod)
+
+import six
+
+from .exception import (MissingAccountData)
```

Comment:
Yes.  They're unnecessary.  I'm undecided whether I like the consistency of always using parens on `from ... import ...` or not.  If you don't like them, I'm OK with not using them.

I was getting tired of adding and removing them as the import lists got bigger and smaller.